### PR TITLE
get_size for cloud stores

### PR DESCRIPTION
**Pull request by xbrianh**

### CODE REVIEW

File: cloud_blobstore/gs.py [hunk 1]
```diff
@@ -114,6 +114,25 @@ def get_user_metadata(
...
 
+    def get_size(
+            self,
+            bucket: str,
+            object_name: str
+    ) -> int:
+        """
+        Retrieves the filesize
+        :param bucket: the bucket the object resides in.
+        :param object_name: the name of the object for which metadata is being
```
Comment (ttung) [inline cloud_blobstore/gs.py]: size

File: cloud_blobstore/__init__.py [hunk 2]
```diff
@@ -87,6 +87,20 @@ def get_user_metadata(
...
 
+    def get_size(
+            self,
+            bucket: str,
+            object_name: str
+    ) -> int:
+        """
+        Retrieves the filesize
+        :param bucket: the bucket the object resides in.
+        :param object_name: the name of the object for which metadata is being
```
Comment (ttung) [inline cloud_blobstore/__init__.py]: size

File: cloud_blobstore/s3.py [hunk 3]
```diff
@@ -197,6 +197,28 @@ def copy(
...
 
+    def get_size(
+            self,
+            bucket: str,
+            object_name: str
+    ) -> int:
+        """
+        Retrieves the filesize
+        :param bucket: the bucket the object resides in.
+        :param object_name: the name of the object for which metadata is being
```
Comment (ttung) [inline cloud_blobstore/s3.py]: size

**Review by ttung — APPROVED** [review]
Can you fix the copy paste?

File: cloud_blobstore/__init__.py (hunk 2 above)
Comment (ttung) [reply cloud_blobstore/__init__.py]: I think size or filesize is fine for "Retrieves the filesize".

I meant the line "the name of the object for which *metadata* is being retrieved" should be updated.

Comment (xbrianh) [reply cloud_blobstore/__init__.py]: Ahhh... miscalibrated eyes I guess

**Pull request merged**